geojson_pydantic/types.py

```diff
@@ -44,13 +45,8 @@ def validate_bbox(bbox: Optional[BBox]) -> Optional[BBox]:
 Position = Union[Tuple[float, float], Tuple[float, float, float]]
 
 # Coordinate arrays
-if TYPE_CHECKING:
-    LineStringCoords = List[Position]
-    LinearRing = List[Position]
-else:
-    LineStringCoords = conlist(Position, min_length=2)
-    LinearRing = conlist(Position, min_length=4)
-
+LineStringCoords = Annotated[List[Position], Field(min_length=2)]
```

Comment:
I did this while tracking down a complaint from mypy and didn't completely mean to have it in this PR. Could revert it and separate it out. It does kind of add `typing-extensions` as a requirement. But `pydantic` includes that requirement already. So I don't know if its completely necessary?

Reply:
I think it's fine 👍